lib/utility/errors.rb

```diff
@@ -32,6 +32,8 @@ def to_h
   end
 
   class ClientError < StandardError; end
+
+  class InvalidFilterConfigError < ClientError; end
```

Comment:
Why inherit from ClientError and not from StandardError?

Reply:
I thought that an invalid filter config can only come from someone typing in an invalid config in the client. But also happy to update it to extend from StandardError.

Reply:
Typically, `ClientError` would be for expressing "I expected a 200, but got a 4xx or 5xx error". So you might have something like:

```
class NotFoundError < ClientError; end
```

to specifically cover 404s. But I expect that `InvalidFilterConfigError` won't have that kind of relationship. So yes, I think inheriting from StandardError is probably better unless I've misunderstood something here.

Reply:
Makes sense, updated it 👍